modules/ROOT/pages/websockets/websockets-connector-server-components.adoc

```diff
@@ -42,16 +42,16 @@ Clients can also indicate the default groups that this endpoint automatically su
 
 == Send a Response
 
-Unlike the `<http:listener>` element which automatically sends a response when the flow ends, this component  never automatically sends a response. This is because the web sockets protocol is full duplex and therefore provides no guarantee that you actually want or need to send a response at all. Depending on your use case, you might want to simply process the messages without sending a response, or only send responses on certain cases.
+Unlike the `<http:listener>` element that automatically sends a response when a flow ends, this component  never automatically sends a response. This is because the web sockets protocol is full duplex and therefore provides no guarantee that you need to or want to send a response. Depending on your use case, you might want to process the messages without sending a response, or only send responses on certain cases.
```

Comment:
```suggestion
Unlike the `<http:listener>` element that automatically sends a response when a flow ends, this component  never automatically sends a response. This is because the WebSocket protocol is full duplex and therefore provides no guarantee that you need to or want to send a response. Depending on your use case, you might want to process the messages without sending a response, or send responses only on certain cases.
```